osquery/tables/system/windows/logical_drives.cpp

```diff
@@ -36,45 +51,20 @@ QueryData genLogicalDrives(QueryContext& context) {
       r["size"] = "-1";
     }
 
+    std::string bootPath = deviceId + "\\Windows";
```

Comment:
Note: We could get rid of this concatenation by changing the `bootDirectories` set into a `std::set<char>` (via `PathGetDriveNumberA`) and doing `bootDirectories.count(deviceId.at(0))`, but I figured that this was slightly easier to read. Thoughts?

Reply:
I think it would be easier to read it with `bootDeviceIds` (with `char`) rather than with real path. Just because impl with device ids gonna be a bit simpler. You can also, if you want, keep `std::strings` instead of `char` and store not only character of device but colon and slashes (`C:\\`)  - it will not be a big issue in terms of performance because of short string optimisation.